app/src/main/java/com/concordia/mcga/activities/SplashActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.concordia.mcga.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+
+public class SplashActivity extends AppCompatActivity {
```

Comment:
Pulled from Master and pushed. Maybe that's what happened? I didn't know it would register as my change.